enhancements/network/dpu/overview.md

```diff
@@ -0,0 +1,269 @@
+software stack on all of the NICs in the cluster. This software will
+need to be kept in sync with the software on the host PC (eg, as new
+functionality is added to OCP). The obvious answer to this problem is
+to run OpenShift on the NIC as well as on the host.
+
+The initial target of this work is the [NVIDIA BlueField-2 DPU].
+However, similar products are being worked on by other vendors which
+we will eventually want to support, so the solution should not be too
+specialized to this one product.
+
+[NVIDIA BlueField-2 DPU]: https://www.nvidia.com/en-us/networking/products/data-processing-unit/
+
+## Glossary
+
+- **Smart NIC** - a vaguely-defined term for an enterprise-class NIC
+  with extensive offload capabilities, but not necessarily with the
+  ability to run arbitrary user software. For example, the NVIDIA
+  Mellanox ConnectX-5 and ConnectX-6.
+
+- **DPU** / **Data Processing Unit** - a Smart NIC with a full CPU,
+  RAM, storage, etc, running a full operating system, able to offload
+  network processing and potentially other tasks from its host system.
+  For example, the NVIDIA BlueField-2.
+
+  (Some vendors other than NVIDIA are using the terms "Smart NIC" and
+  "DPU" slightly differently, but for now at least, this is the way we
+  are defining these terms.)
+
+- **DPU NIC** - refers to the DPU and its CPU, OS, etc (as opposed to
+  the CPU, OS, etc, of the x86 server that the DPU is installed in). A
+  **DPU NIC node** or **DPU NIC worker** is an OCP worker node
+  running on a DPU NIC.
+
+- **DPU Host** - refers to an x86 server (and its CPU, OS, etc) which
+  contains a DPU. A **DPU Host node** or **DPU Host worker** is an OCP
+  worker node running on a DPU Host.
+
+- **Two-Cluster Design** - any architecture for deploying an OCP
+  cluster with (some) nodes containing DPUs, where the DPUs are nodes
+  in a second OCP cluster. (For purposes of this term, the
+  architecture still counts as "two-cluster" even if HyperShift is
+  involved and there are actually three clusters.)
+
+- **Infra Cluster** - in a Two-Cluster Design, the OCP cluster which
+  contains the DPU NIC workers. (It may also include master nodes
+  and/or non-DPU workers, depending on the particular details of the
+  two-cluster design.)
+
+- **Tenant Cluster** - in a Two-Cluster Design, the OCP cluster
+  containing the DPU Host workers. (It may also include master nodes
+  and/or workers without DPUs.)
+
+## Motivation
+
+### User Stories
+
+#### Network Offload
+
+As a cluster administrator, I would like to offload network processing
+from the host CPU to the NIC, to increase throughput and to free up
+host CPU cycles for workloads rather than network processing.
+
+In particular:
+
+- We can move all of the node-side OVS, OVN, and OVN Kubernetes
+  daemons from the host CPU to the NIC CPU. (The host side would
+  retain only a CNI plugin binary.)
+
+- We can move IPsec encryption from the host CPU to the NIC CPU
```

Comment:
Since NIC CPU is not defined, I assume NIC CPU refers to the CPU on the NIC. It's important to note that it is much slower than the host CPU. At the same time, this section mentions an increase in throughput as being one of the goals. For that reason, it makes sense to explicitly add that the real goal of moving IPsec encryption to the NIC CPU is to, in turn, offloaded it to the NIC itself.